include/sir/platform.h

```diff
@@ -82,6 +82,10 @@
 #    endif
 #   endif
 #  endif
+#  if defined(__IMPORTC__)
```

Comment:
"ImportC is an implementation of ISO/IEC 9899:2011, which will be referred to as C11; ImportC is a C compiler embedded into the D implementation. It enables direct importation of C files, without needing to manually prepare a D file corresponding to the declarations in the C file. It directly compiles C files into modules that can be linked in with D code to form an executable. It can be used as a C compiler to compile and link 100% C programs."

TL;DR - It's a C11 compiler included with the the Dlang compiler. =)